pkg/gardenlet/controller/shoot/shoot_control_delete.go

```diff
@@ -469,7 +469,7 @@ func (c *Controller) runDeleteShootFlow(ctx context.Context, o *operation.Operat
 
 	if err := f.Run(flow.Opts{
 		Logger:           o.Logger,
-		ProgressReporter: o.ReportShootProgress,
+		ProgressReporter: flow.NewDelayingProgressReporter(o.ReportShootProgress, c.progressReportFrequency()),
```

Comment:
Can you use the immediate progress reporter here, if the operator did not configure a delay?